erdpy/accounts.py

```diff
@@ -34,8 +35,11 @@ def __init__(self,
             secret_key, pubkey = pem.parse(Path(self.pem_file), self.pem_index)
             self.secret_key = secret_key.hex()
             self.address = Address(pubkey)
-        elif key_file and pass_file:
-            password = get_password(pass_file)
+        elif key_file:
```

Comment:
Ideally, we should not open files or ask for input in the `Account` class. Indeed, currently we read and parse some files here, directly in the constructor (unfortunately).

However, how do you feel about removing the `pass_file` argument of the constructor, and pass a `password: optional string`, instead? (breaking change, but an improvement)

(see comment below)